cla_backend/apps/call_centre/views.py

```diff
@@ -484,9 +485,25 @@ class UserViewSet(CallCentrePermissionsViewSetMixin, BaseUserViewSet):
     filter_backends = (DjangoFilterBackend,)
     filter_fields = ("is_manager",)
 
+    def get_queryset(self):
+        qs = super(BaseUserViewSet, self).get_queryset()
+        operator = self.get_logged_in_user_model()
+        if operator.organisation:
+            query = Q(organisation__isnull=True)
+            query.add(Q(organisation=operator.organisation.id), Q.OR)
+            qs = qs.filter(query)
+        return qs
+
     def get_logged_in_user_model(self):
         return self.request.user.operator
 
+    @transaction.atomic
+    def create(self, request, *args, **kwargs):
+        operator = self.get_logged_in_user_model()
+        if operator.organisation:
+            request.DATA["organisation"] = operator.organisation.id
```

Comment:
Does this have test coverage / has it been fully tested manually?

If you'd made me guess, I'd have said that `DATA` there might have been immutable, but as long as the behaviour's been tested I'm more than happy to concede my guess is wrong.

Reply:
I've added the missing test coverage `call_centre.tests.api.test_user_api.UserTestCase.test_operator_manager_can_create_operator_of_same_organisation`

>DATA there might have been immutable

I think it was created with `mutable=True` https://docs.djangoproject.com/en/1.7/ref/request-response/#django.http.QueryDict.__init__